var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -120,6 +121,12 @@ class Cp2k(MakefilePackage, CudaPackage, CMakePackage, ROCmPackage):
             when="@:7",  # req in CP2K v8+
             description=("Use CUBLAS for general matrix operations in DBCSR"),
         )
+        variant(
+            "cusolvermp",
+            default=True,
```

Comment:
The support of cuSOLVERMp is marked as experimental in 2023.2. Is this the correct default?